kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/attachmentview/KmDocumentView.java

```diff
@@ -330,6 +330,25 @@ public void onAction(boolean didGrant) {
             }
         });
 
+        mainLayout.setOnClickListener(new View.OnClickListener() {
+            public void onClick(View v) {
+                if (!isDownloadRequire()) { return; } // The following code will not be executed if the file is already downloaded.
+                if (kmStoragePermissionListener.isPermissionGranted()) {
+                    playAudio();
```

Comment:
Why the method name is playAudio? What exactly it does with attachments like PDF

Reply:
All the attachment files and audio files are using the same function to preview the file or play the audio.

Reply:
As discussed change the name accordingly in future checkins